src/Item.php

```diff
@@ -161,66 +161,12 @@ public function getRelationships(): array
         return $relationships;
     }
 
-    /**
-     * @TODO: MEGA TODO. Set up a serializer for the Item so that we can remove this, getRelationships etc
-     *
-     * @return \Swis\JsonApi\Client\Collection
-     */
-    public function getIncluded(): Collection
-    {
-        $included = new Collection();
-
-        foreach ($this->relationships as $name => $relationship) {
-            if ($relationship->shouldOmitIncluded() || !$relationship->hasIncluded()) {
-                continue;
-            }
-
-            if ($relationship instanceof OneRelationInterface) {
-                /** @var \Swis\JsonApi\Client\Interfaces\ItemInterface $item */
-                $item = $relationship->getIncluded();
-                if ($item->canBeIncluded()) {
-                    $included->push($item->toJsonApiArray());
-                }
-                $included = $included->merge($item->getIncluded());
-            } elseif ($relationship instanceof ManyRelationInterface) {
-                $relationship->getIncluded()->each(
-                    function (ItemInterface $item) use (&$included) {
-                        if ($item->canBeIncluded()) {
-                            $included->push($item->toJsonApiArray());
-                        }
-                        $included = $included->merge($item->getIncluded());
-                    }
-                );
-            }
-        }
-
-        return $included
-            ->unique(
-                function (array $item) {
-                    return $item['type'].':'.$item['id'];
-                }
-            )
-            ->values();
-    }
-
     /**
      * @return bool
      */
-    public function canBeIncluded(): bool
+    public function hasRelationships(): bool
     {
-        if (empty($this->getType())) {
-            return false;
-        }
-
-        if (null === $this->getId()) {
-            return false;
-        }
-
-        if (empty($this->relationships) && empty($this->toArray())) {
-            return false;
-        }
-
-        return true;
+        return !empty($this->getRelationships());
```

Comment:
Isnt empty a bit of an overkill here? Just `count($this->getRelationships())` would be enough.

Reply:
`empty` is generally considered faster/better than `count` for an array